src/server/pachyderm_test.go

```diff
@@ -1169,7 +1169,26 @@ func TestSimple(t *testing.T) {
 	require.Equal(t, "foo\nfoo\n", buffer.String())
 }
 
-func TestPipelineWithMultipleInputs(t *testing.T) {
+func TestPipelineWithDifferentInputMethods1(t *testing.T) {
```

Comment:
nit: Instead of numbering these 1-4, what if we put the parameters in the test name? e.g. TestPipelineInputMapMap, TestPipelineInputReduceMap, TestPipelineInputReduce_RepoDiff, TestPipelineInputReduceReduce? (or something like that)